pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -188,7 +228,20 @@ func (p *Plugin) Attest(stream nodeattestorv1.NodeAttestor_AttestServer) error {
 		return status.Errorf(codes.PermissionDenied, "challenge response verification failed: %v", err)
 	}
 
-	spiffeid, err := x509pop.MakeAgentID(config.trustDomain, config.pathTemplate, leaf)
+	SVIDPath := ""
+	if config.SPIRETrustBundle {
+		if len(leaf.URIs) >= 1 {
```

Comment:
nit: if you invert the condition, you can do the early failure return when there are no URIs and then not need an else (reduces indentation)